### DISCUSSION TITLE
arg to point to valid credentials?

### DISCUSSION THREAD

**Discussion by 88gts** [tdorssers/TeslaPy#83] [Q&A]
hey all, i wrote a bash script that requests my vehicles status but it is failing as it can't find a valid credential in the working directory.  note that this works in my other workflows, the credential is valid and working.  is there a way to pass in the location of the credential (which i assume is cache.json?) to work around my issues?  thanks!

from script 
```
cat tesla.log 
Use browser to login. Page Not Found will be shown at success.
Open this URL to authenticate: XXX
```

proof of working credential
{
    "gps_as_of": 1656371072,
    "heading": 257,
    "latitude":  ,
    "longitude": - ,
    "native_latitude":  ,
    "native_location_supported": 1,
    "native_longitude": - ,
    "native_type": "wgs",
    "power": 0,
    "shift_state": null,
    "speed": null,
    "timestamp": 1656373283689
}

**Comment by 88gts** [comment]
as a workaround, i was able to symlink cache.json.

**Comment by tdorssers** [comment]
simply set the cache_file argument of the Tesla class constructor to '/your/path/to/json.cache'
